users/config.rst

```diff
@@ -354,9 +354,12 @@ copyOwnershipFromParent
     Requires running Syncthing as privileged user, or granting it additional capabilities (e.g. CAP_CHOWN on Linux).
 
 modTimeWindowS
-    Allowed modification timestamp difference when comparing files for equivalence.
-    To be used on systems that have unstable modification timestamps, that might change after being observed after
-    the last write operation. Used in Android only.
+    Allowed modification timestamp difference when comparing files for
+    equivalence. To be used on file systems which have unstable or
+    imprecise modification timestamps (e.g. FAT) that might change after
+    being recorded during the last write operation. Must be set before
+    adding files to the folder in order to take effect. Usually defaults
```

Comment:
I know the context of this addition, but I don't think "must be set before adding files to the folder to take effect" increases understanding much here... Any config change only affects what happens after it is made, obviously, but this somehow implies it won't work if set on an existing folder.

Perhaps something in the first sentence about the comparison being used to decide whether a file has changed and needs to be rehashed? Then it might be more clear that changing this option doesn't un-rehash and un-change the file...

Reply:
Yeah, I know that the sentence is vague. I could not come up with anything better.

What do you think about something like this?

```Must be set before adding files to the folder in order to take effect on those files.```

Otherwise I will try to think about how to rewrite the sentences to explain the point better.

Reply:
I don't particularly like that either, because it makes it sound like the value is somehow locked in per file once we've seen it the first time... while in reality it simply applies each and every time we look at a file and consider the modification timestamp.

Reply:
I reworded the explanation a little bit, and also removed the controversial line. I agree that it will be better to explain this behaviour with regards to RO folders in general rather than having separate explanations under each setting.